fabric-convention-tags-v1/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/BiomeTagGenerator.java

```diff
@@ -90,6 +90,10 @@ private void generateCategoryTags() {
 		getOrCreateTagBuilder(ConventionalBiomeTags.EXTREME_HILLS)
 				.add(BiomeKeys.WINDSWEPT_GRAVELLY_HILLS)
 				.add(BiomeKeys.WINDSWEPT_HILLS);
+		getOrCreateTagBuilder(ConventionalBiomeTags.WINDSWEPT_HILLS)
+				.addOptionalTag(ConventionalBiomeTags.EXTREME_HILLS)
```

Comment:
If we go for this tag, I would rather list them than include the tag - mods might add extreme hills that are not windswept?